server/channels/store/sqlstore/preference_store.go

```diff
@@ -315,3 +315,52 @@ func (s SqlPreferenceStore) CleanupFlagsBatch(limit int64) (int64, error) {
 
 	return rowsAffected, nil
 }
+
+func (s SqlPreferenceStore) DeleteInvalidVisibleDmsGms() (int64, error) {
+	var query string
+	if s.DriverName() == "postgres" {
+		query = `DELETE FROM Preferences
```

Comment:
Instead of deleting these, could we just update them to them to have a value of 40? I think the default for these is 10, so it'll be a bit less jarring for affected users to go from unlimited to 40 instead of to 10

Reply:
The default is 40 on the frontend so we are ok to delete these